Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -295,3 +380,19 @@ extension SecureEnclaveValet {
     }
 
 }
+
+// MARK: - Private Extensions
+
+private extension SecureEnclaveAccessControl {
+
+    /// The `LAPolicy` that the `SecureEnclaveAccessControl` corresponds to.
+    var policy: LAPolicy {
```

Comment:
Paired with the suggestion above:
```suggestion
    private var policy: LAPolicy {
```

Reply:
👍